### PR TITLE
[v9.1.x] ReleaseNotes: Updated changelog and release notes for 9.1.7 (#56313)

### PR DESCRIPTION
**Pull request by papagian**
(cherry picked from commit a50017da1d60bc45e95f172a96bf87bc44973a6c)

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,35 @@
...
+- **Alerting:** Skip unsupported file types on provisioning. [#55573](https://github.com/grafana/grafana/pull/55573), [@JohnnyQQQQ](https://github.com/JohnnyQQQQ)
+- **AzureMonitor:** Ensure resourceURI template variable is migrated. [#56095](https://github.com/grafana/grafana/pull/56095), [@aangelisc](https://github.com/aangelisc)
+- **Dashboard:** Fix plugin dashboard save as button. [#55197](https://github.com/grafana/grafana/pull/55197), [@lpskdl](https://github.com/lpskdl)
+- **Docs:** Fix decimals: auto docs for panel edit. [#55477](https://github.com/grafana/grafana/pull/55477), [@joshhunt](https://github.com/joshhunt)
+- **Fix:** RBAC handle `error no resolver` found. [#55676](https://github.com/grafana/grafana/pull/55676), [@gamab](https://github.com/gamab)
+- **Fix:** RBAC handle `error no resolver` found. (Enterprise)
+- **LibraryPanelSearch:** Refactor and fix hyphen issue. [#55314](https://github.com/grafana/grafana/pull/55314), [@kaydelaney](https://github.com/kaydelaney)
+- **Live:** Fix live streaming with `live-service-web-worker` feature flag enabled. [#55558](https://github.com/grafana/grafana/pull/55558), [@grafanabot](https://github.com/grafanabot)
```
Comment (gguillotte-grafana) [inline CHANGELOG.md:28]: This is a duplicate line attributed to grafanabot and should be removed.

File: docs/sources/release-notes/release-notes-9-1-7.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
++++
+title = "Release notes for Grafana 9.1.7"
+hide_menu = true
++++
```
Comment (gguillotte-grafana) [inline docs/sources/release-notes/release-notes-9-1-7.md:4]: Use YAML over TOML for Hugo frontmatter per @jdbaldry's guidance:

```suggestion
---
title: Release notes for Grafana 9.1.7
hide_menu: true
---
```

File: docs/sources/release-notes/release-notes-9-1-7.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+- **Alerting:** Skip unsupported file types on provisioning. [#55573](https://github.com/grafana/grafana/pull/55573), [@JohnnyQQQQ](https://github.com/JohnnyQQQQ)
+- **AzureMonitor:** Ensure resourceURI template variable is migrated. [#56095](https://github.com/grafana/grafana/pull/56095), [@aangelisc](https://github.com/aangelisc)
+- **Dashboard:** Fix plugin dashboard save as button. [#55197](https://github.com/grafana/grafana/pull/55197), [@lpskdl](https://github.com/lpskdl)
+- **Docs:** Fix decimals: auto docs for panel edit. [#55477](https://github.com/grafana/grafana/pull/55477), [@joshhunt](https://github.com/joshhunt)
+- **Fix:** RBAC handle `error no resolver` found. [#55676](https://github.com/grafana/grafana/pull/55676), [@gamab](https://github.com/gamab)
+- **Fix:** RBAC handle `error no resolver` found. (Enterprise)
+- **LibraryPanelSearch:** Refactor and fix hyphen issue. [#55314](https://github.com/grafana/grafana/pull/55314), [@kaydelaney](https://github.com/kaydelaney)
+- **Live:** Fix live streaming with `live-service-web-worker` feature flag enabled. [#55558](https://github.com/grafana/grafana/pull/55558), [@grafanabot](https://github.com/grafanabot)
```
Comment (gguillotte-grafana) [inline docs/sources/release-notes/release-notes-9-1-7.md:33]: Remove duplicate line (see first comment)

**Review by gguillotte-grafana — CHANGES_REQUESTED** [review]
There's a PR credited twice, once to grafanabot. Otherwise, looks great!

**Review by gguillotte-grafana — APPROVED** [review]
This is a backport PR - I'll file a PR upstream to rectify the review's issues.

**Comment by papagian** [comment]
@gguillotte-grafana the upstream PR is automatically created by this GH [action](https://github.com/grafana/grafana/actions/workflows/update-changelog.yml). So maybe we have to update [the source](https://github.com/grafana/grafana-github-actions/tree/main/update-changelog)

**Pull request merged**